geth/jail/internal/loop/loop_test.go

```diff
@@ -0,0 +1,56 @@
+package loop_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/status-im/status-go/geth/jail/internal/loop/looptask"
+
+	"github.com/robertkrimen/otto"
+	"github.com/status-im/status-go/geth/jail/internal/loop"
+	"github.com/status-im/status-go/geth/jail/internal/vm"
+	"github.com/stretchr/testify/suite"
+)
+
+func (s *LoopSuite) TestAddAndReady() {
```

Comment:
It isn't looks like unit test more like e2e.

Reply:
I am testing that errors are returned appropriately. Should I do this a different way?